packages/invoices/src/InboundShipment/api/hooks.ts

```diff
@@ -269,3 +294,43 @@ export const useDeleteSelectedInbounds = () => {
 
   return deleteAction;
 };
+
+export const useInboundRows = () => {
```

Comment:
Same as outbound - single hook for getting either the lines or items depending on whether grouped or not.